HealthcareSystem/Backend/Model/Users/Patient.cs

```diff
@@ -16,19 +16,19 @@ public class Patient : User
     {
         [DatabaseGenerated(DatabaseGeneratedOption.Identity)]
         public DateTime DateOfRegistration { get; set; }
-        public bool IsGuest { get; set; }
```

Comment:
Why remove this? Are guest accounts no longer a thing in our system or?

Reply:
the guest no longer exists in the system